File: 1700-number-of-students-unable-to-eat-lunch/1700-number-of-students-unable-to-eat-lunch.cpp
class Solution {
public:
    int countStudents(vector<int>& st, vector<int>& sw) {
        int a[2];
        for(auto it:st) a[it]++;
        for(auto it:sw){
            if(a[it]) a[it]--;
            else break;
        }
        return a[0]+a[1];
    }
};

File: 1700-number-of-students-unable-to-eat-lunch/1700-number-of-students-unable-to-eat-lunch.py
class Solution:
    def countStudents(self, students: List[int], sandwiches: List[int]) -> int:
        zero,one=0,0
        for s in students:
            if s==0:
                zero+=1
            else:
                one+=1
        for s in sandwiches:
            if s==0 and zero:
                zero-=1
            elif s==1 and one:
                one-=1
            else :
                return zero+one
        return zero+one